targets/UnrealMarketplacePlugin/make.js

```diff
@@ -53,9 +55,23 @@ exports.makeCombinedAPI = function (apis, sourceDir, baseApiOutputDir) {
         // Copy the PlayFabPlugin folder just created into the ExampleProject
         // TODO: It causes very confusing problems to copy from an output subdir to another output subdir. Let's fix this
         templatizeTree(locals, path.resolve(apiOutputDir, "PlayFabPlugin"), path.resolve(apiOutputDir, "ExampleProject/Plugins"));
+
+        // Create a Content folder that can hold testTitleJSON, so that UE4 can handle putting it onto mobile devices in particular
+        // This works in conjunction with a line in DefaultGame.ini
+        var jsonFolderPath = path.resolve(apiOutputDir, "ExampleProject", "Content", "JSON");
+        fs.mkdir(jsonFolderPath, { recursive: true }, handleError);
+
+        // Don't require PF_TEST_TITLE_DATA_JSON during SDK generation - some devs don't need it, or might just want to put it into the folder manually
+        if (testTitleJsonPath !== undefined) {
+            fs.copyFile(testTitleJsonPath, path.join(jsonFolderPath, "testTitleData.json"), handleError);
```

Comment:
Definitely don't do this.
We don't want to accidentally leak any secure testTitleData files.

Reply:
Even after the .gitIgnore changes, we can't do this as part of SdkGenerator.
I'm not sure if this is ever safe to do in an automated way.
I will think about it, but we're definitely not doing this here.

Reply:
I think the one danger here is if a developer (who is set up with testTitleData.json) doesn't know that this functionality exists, and then decides to copy-paste their ExampleProject to somewhere public.  That could be solved by moving this functionality into a dedicated command script - if you have to click "copy testTitleData to test runner projects", then at that point I think it's equal risk to not having this code at all (i.e. requiring developers to manually copy/paste the file into every test project), but still less tedious.